### PR TITLE
Completed parts 1 - 4. Couldn't get any further.

### PR DESCRIPTION
**Pull request by austie702**

### CODE REVIEW

File: queries.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Character.objects.filter(inventory_isnull=False).count()
+Character.objects.filter(inventory_isnull=False)
+all_items = Character.objects.filter(inventory_isnull=False).count()
+all_items = Character.objects.filter(inventory_isnull=True).count()
+Item.objects.all().count()
+Item.objects.all().name()
+Item.objects.all().average()
+Item.objects.get(id)
+Item.objects.in_bulk()
+# I honestly don't know how to do parts 5 & 6. I'm going to push and stop and wait for the solution lecture.
```
Comment (JCamp-12) [inline queries.py:47]: good job getting through what you know!

**Review by JCamp-12 — COMMENTED** [review]
Austin, great working with you.  Keep pushing and best of luck on your project,  Thanks!

File: queries.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Item.objects.all().count()
+
+
+# 04a: Number of Weapon Items: 37
+Weapon.objects.all().count()
+# 04b. Number of Non-Weapon Items: 137
+###### Subtracted
+
+
+# 05. Average number of Items each Character has:
```
Comment (TheDeterminator) [inline queries.py:35]: You may be able to solve this using the built-in django .aggregate and the Avg method from django.db.models but aside from that the more straightforward solution is Avg items per character = Num of items/num characters or in other words Character.objects.all().count()/Item.objects.all().count(). I imagine you probably have the answers now from the solution lecture though.

**Review by TheDeterminator — COMMENTED** [review]
Austin, good job with this. Please see my sprint review for more in-depth feedback.

**Pull request closed without merging**